docs/requirements.txt

```diff
@@ -2,3 +2,5 @@ pyenchant
 Sphinx>=4.5.0
 sphinxcontrib-spelling
 blacken-docs
+# The black version needs to match the one from .pre-commit-config.yaml
+black==24.2.0
```

Comment:
Please don't do this. It's enough that we have to bump versions in `.pre-commit-config.yaml`. Also, there is no difference between `24.2.0` and `24.4.2`.

Reply:
I agree that this change has no effect right now, but if we leave `black` unpinned in CI then it _might_ cause the same error I experienced in the future where the CI check and the local pre-commit diverge.

If you think the tradeoff is worth it I'm happy to drop the commit and only keep the one that adds `--rst-literal-block`.

Reply:
Please keep the `args: ["--rst-literal-block"]` change and drop the comments 👍